### PR TITLE
Update verification protocol for v1.5

### PR DESCRIPTION
**Pull request by mikehelmick**
* replace test date with symptom onset date
* populate new claims from the exposure notification server claim set

### CODE REVIEW

File: pkg/controller/certapi/certificate.go [hunk 1]
```diff
@@ -152,8 +156,13 @@ func (ca *CertificateAPI) ServeHTTP(w http.ResponseWriter, r *http.Request) {
...
 	claims := verifyapi.NewVerificationClaims()
-	// This server curently does not provide any transmission risk overrides, although that is part
-	// of the diagnosis verification protocol.
+	// Assign the report type.
+	claims.ReportType = subject.TestType
+	if subject.SymptomDate != nil {
+		claims.SymptomOnsetInterval = subject.SymptomInterval()
+	}
+
+	// TODO(mikehelmick): Assign transmission risk overrides. Algorithm not set yet.
```
Comment (crwilcox) [inline pkg/controller/certapi/certificate.go:165]: is this planned soon, or should we make a tracking issue?

File: pkg/database/migrations.go [hunk 2]
```diff
@@ -118,6 +118,43 @@ func (db *Database) RunMigrations(ctx context.Context) error {
 				return nil
 			},
 		},
+		{
+			ID: "00007-AddSymptomOnset",
+			Migrate: func(tx *gorm.DB) error {
+				logger.Info("db migrations: rename test_date to symptom_date")
+				// AutoMigate will add missing fields.
```
Comment (crwilcox) [inline pkg/database/migrations.go:125]:
```suggestion
				// AutoMigrate will add missing fields.
```

File: cmd/server/assets/home.html [hunk 3]
```diff
@@ -47,13 +47,13 @@ <h1>Issue Diagnosis Verification Code</h1>
...
-        <label for="testDate" class="col-sm-3">Test date</label>
+        <label for="symptomDate" class="col-sm-3">Symptom Onset Date</label>
         <div class="col-sm-9">
           <div class="datepicker date input-group p-0 shadow-sm">
-            <input type="date" id="testDate" name="testDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" />
+            <input type="date" id="symptomDate" name="symptomDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" />
           </div>
           <small class="form-text text-muted">
-            Optional, cannot be more than 14 days prior to today. This system is based on
+            <strong>Recommended</strong>: cannot be more than 14 days prior to today. This system is based on
```
Comment (crwilcox) [inline cmd/server/assets/home.html:56]: is this actually 'cannot'? Do we prevent a user from configuring it less?

File: pkg/controller/certapi/certificate.go (hunk 1 above)
Comment (mikehelmick) [reply pkg/controller/certapi/certificate.go:165]: actively working on it - part of this same issue.

File: cmd/server/assets/home.html (hunk 3 above)
Comment (mikehelmick) [reply cmd/server/assets/home.html]: fixed - this now shows the configured value.

**Comment by mikehelmick** [comment]
rebased on main

**Pull request merged**